aos/aos_milestones/IMpg/check_impg_run.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>Should we put these in a separate place? (I already do for my notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Yes, but we need to find a good place for them... </p>